tutorial/numerical-entry.md

```diff
@@ -124,4 +137,5 @@ func main() {
 	w.SetContent(entry)
 	w.ShowAndRun()
 }
+
```

Comment:
Can probably lose this newline addition.

Reply:
Yup. Removed now.